provisionersdk/proto/provisioner.proto

```diff
@@ -232,6 +232,7 @@ message Metadata {
     string workspace_owner_ssh_public_key = 15;
     string workspace_owner_ssh_private_key = 16;
     string workspace_build_id = 17;
+	string workspace_owner_login_type =  18;
```

Comment:
Nit: formatting

Reply:
What's the exact formatting problem here?

Reply:
Tabs vs spaces, it seems